Saule/Http/NoDefaultIncludedAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+
+namespace Saule.Http
+{
+    /// <summary>
+    /// Indicates that the action should not include related data by default.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public sealed class NoDefaultIncludedAttribute : ActionFilterAttribute
```

Comment:
I'm not entirely sure what you mean. You mean to say that allowing querying should implicitly allow the include param (or the other way round) and at the same time disable the default include? I think something like that might make sense.